solana-programs/anchor/audius-data/programs/audius-data/src/lib.rs

```diff
@@ -1066,9 +1064,9 @@ pub enum EntityTypes {
     Playlist,
 }
 
-// Seed & bump used to validate the user's handle with the account base
+// Seed & bump used to validate the user's id with the account base
 #[derive(AnchorSerialize, AnchorDeserialize, Clone, PartialEq)]
-pub struct UserHandle {
+pub struct UserSeedBump {
```

Comment:
changes look really good so far. my 2 cents - I think we should store "seed" (maybe we call it ID) as a `u32` type and call `to_le_bytes` before passing it in https://doc.rust-lang.org/std/primitive.u32.html#method.to_le_bytes. `u32` gives a maxvalue of 4294967295 and postgres Integer max is 2147483648

Reply:
@hareeshnagaraj what if we do the conversion before we construct `UserSeedBump` so we don't have to dupe the conversion bit everywhere it's passed in? is it bc this changes compute/storage costs?

Reply:
its a bit clearer i think to have a numeric ID instead of bytes